src/libs/data-utils.js

```diff
@@ -0,0 +1,121 @@
+import _ from 'lodash/fp'
+import filesize from 'filesize'
+import { Fragment } from 'react'
+import { code, div, h, pre } from 'react-hyperscript-helpers/lib/index'
+import Collapse from 'src/components/Collapse'
+import { buttonPrimary, link } from 'src/components/common'
+import { spinner } from 'src/components/icons'
+import Modal from 'src/components/Modal'
+import { TextCell } from 'src/components/table'
+import { Buckets, Martha } from 'src/libs/ajax'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+
+
+const styles = {
+  cell: children => div({ style: { marginBottom: '0.5rem' } }, children),
```

Comment:
This seems misplaced to me because it's effectively a component, not a style. I think the abstraction is fine, but perhaps it should be a standalone function for clarity.